tests/conftest.py

```diff
@@ -30,16 +32,26 @@ def comparison_dirs(tmp_path_factory, golden_dirs):
         comparison_dirs = []
         for dir_ in golden_dirs:
             path = Path(dir_)
-            if not path.exists():
-                path.mkdir()
+            assert path.name in ('main', 'develop')
```

Comment:
What's the motivation for limiting the dir names here? I've been in the habit of naming my dirs `prod` and `test`

Reply:
Hmm, I see it's so the `zip()` calls later always correctly pair the right HyP3 session with the corresponding directories. I'd prefer if things weren't coupled that tightly, but I'm yet to think of a better approach.

Reply:
refactored to assume comparison dirs are [golden, test] as is already documented in our readme so I don't need this assert, and bundled hyp3 APIs with this fixture (now `comparison_environments`)